src/Abp/Application/Features/IFeatureChecker.cs

```diff
@@ -4,21 +4,21 @@
 namespace Abp.Application.Features
 {
     /// <summary>
-    /// This interface should be used to get value of
+    /// This interface should be implemented to get the value of features
```

Comment:
"should be implemented" may lead to misunderstanding, because it's already implemented in Abp.Zero packages and it has alrady an implementation for most developers. They will think they should implement it.